charts/rancher-turtles/Chart.yaml

```diff
@@ -13,7 +13,7 @@ keywords:
 - provisioning
 dependencies:
   - name: cluster-api-operator
-    version: v0.16.0
+    version: v0.17.0
```

Comment:
I wonder if https://github.com/rancher/turtles/pull/1105 and https://github.com/rancher/turtles/pull/1106 doesn’t cover this change already, but looks good to merge it combined.

Reply:
I have not noticed them being open, but this can also go as is.